test/higher_order_ops/test_print.py

```diff
@@ -39,6 +41,49 @@ def f(x):
 
         self.assertEqual(printed_output, "moo 1 2")
 
+    def test_tensor_print(self):
```

Comment:
instead of adding this test, in the make_fx test can you run gm(*inputs) and check the correct strings are printed when running that?

Reply:
updated!